src/theory/quantifiers/sygus/sygus_grammar_cons.h

```diff
@@ -63,30 +63,42 @@ class CegGrammarConstructor
  /** is the syntax restricted? */
  bool isSyntaxRestricted() { return d_is_syntax_restricted; }
  /** make the default sygus datatype type corresponding to builtin type range
- *   bvl is the set of free variables to include in the grammar
- *   fun is for naming
- *   extra_cons is a set of extra constant symbols to include in the grammar,
- *   exclude_cons is used to exclude operators from the grammar,
- *   term_irrelevant is a set of terms that should not be included in the
- *      grammar.
- */
+  *   bvl is the set of free variables to include in the grammar
+  *   fun is for naming
+  *   extra_cons is a set of extra constant symbols to include in the grammar,
+  *   regardless of their inclusion in the default grammar.
+  *   exclude_cons is used to exclude operators from the grammar,
+  *   term_irrelevant is a set of terms that should not be included in the
+  *      grammar.
+  *   include_cons is a set of operators such that if this set is not empty,
```

Comment:
Can you clarify this point? Do you mean that if include_cons is non-empty, then it defines the complete set of operators for that grammar?